node/src/proxy_server/mod.rs

```diff
@@ -1300,7 +1301,15 @@ impl Hostname {
         Self { hostname }
     }
 
+    fn is_running_in_integration_test() -> bool {
+        env::var("MASQ_INTEGRATION_TEST").is_ok()
+    }
+
     fn validate_hostname(&self) -> Result<(), String> {
```

Comment:
It just struck me that this could be renames as `validate_host_is_on_wan_rather_than_lan`. I'll leave it up to you guys. I thought something along these lines would be quite a nice name.